### ISSUE TITLE
Dependencies that are executed as a sub-command are not detected.

### ISSUE THREAD

**Issue by ChiefORZ** [webpro/knip#243]
When using knip to detect unused cli commands following example would detect esbuild as correctly being used:
```json
# package.json
{
  "scripts": {
    "build:esbuild": "esbuild src/index.ts --outdir=build",
  }
}
```

```sh
➜ pnpm knip         
Unused dependencies (2)
eslint              package.json
eslint-config-next  package.json
```

But when invoking this command with nx to enable caching, knip lists esbuild as unused dependency:
```json
# package.json
{
  "scripts": {
    "build:esbuild": "nx exec -- esbuild src/index.ts --outdir=build",
  }
}
```

```sh
➜ pnpm knip
Unused dependencies (2)
eslint              package.json
eslint-config-next  package.json

Unused devDependencies (1)
esbuild  package.json
```

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.25.0. See [Release 2.25.0](https://github.com/webpro/knip/releases/tag/2.25.0) for release notes._

**Comment by webpro** [comment]
Thanks @ChiefORZ, added the Nx binary resolver.
